src/docfx/validation/ContentValidator.cs

```diff
@@ -215,6 +219,31 @@ public void ValidateTocEntryDuplicated(FilePath file, List<FilePath> referencedF
             }
         }
 
+        public void ValidateZonePivots(FilePath file, List<(TripleColonBlock, string)> zonePivotUsages)
```

Comment:
nit: Take `SourceInfo<T>` as other methods like `ValidateImageLink`
```suggestion
        public void ValidateZonePivots(FilePath file, List<SourceInfo<string> zonePivotUsages)
```